### PR TITLE
Create mobile version for work section and about me section

### PR DESCRIPTION
**Pull request by mandelatuks**
**The following were added to the mobile version of portfolio**

-Add about me section mobile
-Add work section mobile
-Add style about me and work section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,62 +1,246 @@
...
+        <div>
+          <a class="viewmobile">Welcome
+            <i class="fa-solid fa-bars"></i>
+          </a>
+
+        </div>
+
+      </nav>
+
+
```
Comment (nicupop729) [inline index.html]: In order to have a nice and clean repo, is not recommended to have so many empty lines. :sunglasses: 

- [ ] Please remove empty lines from your HTML file. You can keep if you want some empty lines, but only as separators between big sections such as  `head` and `body`, or  `header` and `main`.

File: index.html [hunk 2]
```diff
@@ -1,62 +1,246 @@
...
+          <button type="button" class="btn-card"> See Project</button>
+        </div>
+        </article>
+      </div>
+
+    </section>
+
+    
+
+    <section class="about_section">
```
Comment (nicupop729) [inline index.html:179]: - [ ] Your background images are not displaying. Please take measures to fix this issue, especially for the about section, as for this milestone you have to implement the work and about section. But It is highly recommended to implement it for the headline too.

File: styles.css [hunk 3]
```diff
@@ -78,7 +78,312 @@
...
+    background-image: url('Images/illustration1-about-me-mobile.svg'), url('Images/illustration2-about-me-mobile.svg');
+    background-position: top right, bottom left;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    padding: 0;
+    width: 351px;
+    height: 333px;
```
Comment (nicupop729) [inline styles.css]: The about me section takes fixed width which is not recommended as we want to implement a responsive design. 

- [ ] Please consider to not use fix width for big items and sections in our design;

Your design: 
image

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @mandelatuks , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Required Changes ♻️

- [ ] Please create a `.gitignore` file, place it on your root path and inside set the `node_modules` directory.
- [ ] As we are using the Mobile-First Design approach and we were not yet introduced to media queries, it is not recommended to use media queries. That's way under 360px your design breaks completely. You set your media queries only for devices above 360px. Please remove the media queries for now.
Your design:
image
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] As your background is not showing for the headline, please take measures to fix it;
- [ ] The logo and media icons are not displaying. Kindly fix them;
- [ ] The logo and the hamburger button are glued to the margins of the screen. It would be better if we apply some margins and maybe padding to our design.
Your design:
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mandelatuks** [comment]
@nicupop729
**the images are screenshots i took for the issue regarding icons and images not responding please see them**


Annotation 2022-03-19 115443
Annotation 2022-03-19 115524

**Review by Mre55 — APPROVED** [review]
**Hi @mandelatuks,**

# Status: APPROVED  🎊 🥇

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**